src/palace/manager/sqlalchemy/model/library.py

```diff
@@ -206,9 +206,26 @@ def associated_collections(self) -> Sequence[Collection]:
 
     @property
     def active_collections(self) -> Sequence[Collection]:
```

Comment:
Minor: It might make sense to try to use a relationship instead of a property for this. I think that would let sqlalchemy handle the caching for us. 

Something like this: 
https://github.com/ThePalaceProject/circulation/blob/81967eba87b897287fee14e87e7b4f474f625a91/src/palace/manager/sqlalchemy/model/licensing.py#L318-L333

Reply:
It doesn't look like we can do this right now because of how the collection subscriptions filters are currently structured. I do think it could be done if we switch to db date functions. I took a quick stab at it, but ran into an import cycle. I'm going to back off on this for now, since I want to continue to focus on the performance issues. I'll make a ticket to try this out in the near future.